robustbench/model_zoo/cifar10.py

```diff
@@ -341,6 +340,21 @@ def forward(self, x):
         return torch.mean(torch.stack(out_list),dim=0)
 
 
+class Modas2021PRIME_ResNet18(ResNet):
```

Comment:
i'd rename it without `_` for the sake of consistency with the existing models (e.g., like [here](https://github.com/RobustBench/robustbench/blob/master/robustbench/model_zoo/cifar10.py#L38)), i.e. `Modas2021PRIME_ResNet18` -> `Modas2021PRIMEResNet18`